{"links": {"self": {"href": "data/repositories/osrf/subt/issues/300/comments/55563473.json"}, "html": {"href": "#!/osrf/subt/issues/300#comment-55563473"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/subt/issues/300.json"}}, "type": "issue", "id": 300, "repository": {"links": {"self": {"href": "data/repositories/osrf/subt.json"}, "html": {"href": "#!/osrf/subt"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3c95f6ad-c304-407c-b838-09597d836552}ts=2272898"}}, "type": "repository", "name": "subt", "full_name": "osrf/subt", "uuid": "{3c95f6ad-c304-407c-b838-09597d836552}"}, "title": "Running into \"max depth exceeded\" error with the current subt-virtual-testbed base image"}, "content": {"raw": "I ran into this as well.  My fix was to reduce the number of RUN commands used in the testbed image \\(by not downloading each ignition file as a separate RUN command\\).", "markup": "markdown", "html": "<p>I ran into this as well.  My fix was to reduce the number of RUN commands used in the testbed image (by not downloading each ignition file as a separate RUN command).</p>", "type": "rendered"}, "created_on": "2020-01-07T20:29:02.822371+00:00", "user": {"display_name": "Neil Johnson", "uuid": "{ace5bb77-1f3f-47ce-ad95-ea205b690930}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bace5bb77-1f3f-47ce-ad95-ea205b690930%7D"}, "html": {"href": "https://bitbucket.org/%7Bace5bb77-1f3f-47ce-ad95-ea205b690930%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/7ff0dae55a4eb43a021cc9c08bbf30f5d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsNJ-3.png"}}, "nickname": "realdealneil1980", "type": "user", "account_id": "557058:21a4b3d0-2fa2-4202-9385-337ffcf40b2f"}, "updated_on": null, "type": "issue_comment", "id": 55563473}